dsgrid/utils/spark_partition.py

```diff
@@ -10,21 +10,37 @@ class SparkPartition:
     def __init__(self):
         return
 
-    def get_data_size(self, df, bytes_per_cell=64):
+    def get_data_size(self, df, bytes_per_cell=8):
         """ approximate dataset size """
```

Comment:
Add a full doc string. I don't know what bytes_per_cell means (and therefore can't follow why the default is being changed from 64 to 8), and right now you have to look at the source code to know what's going to be returned and what the data size units are.